lib/kdump_utils.pm

```diff
@@ -396,7 +396,7 @@ sub configure_service {
     if ($args{yast_interface} eq 'cli') {
         is_transactional ? activate_kdump_transactional : activate_kdump_cli;
     } else {
-        return 16 if activate_kdump == 16;
+        return 16 if activate_kdump(increase_kdump_memory => 0) == 16;
```

Comment:
PR description says `Remove softfail`, but you just added option which skips memory increase without any workaround removal.  Also verification is  done on TW only. Please do VR for all impacted archs ppc64le+aarch64 on all SLE supported version SLE-12 SP5. SLE-15 SP2,3,4,5,6,7 and SLE 16.

Reply:
https://openqa.suse.de/tests/overview?build=tinawang123%2Fos-autoinst-distri-opensuse%23kdump_and_crash
I only found those jobs and rerun them.